Addon/Context/NoFrameworkRenderingContext.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using EPiServer.Core;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using RenderingLayoutProcessor.Extension;
+
+namespace RenderingLayoutProcessor.Context
+{
+    public class NoFrameworkRenderingContext : IRenderingContentAreaContext
```

Comment:
Name this to tell what it does instead?
AttributingRenderingContext?
MetadataAwareRenderingContext?
...